system/storage/afs.go

```diff
@@ -0,0 +1,75 @@
+package storage
+
+import (
+	"context"
+	"github.com/viant/endly"
+	"github.com/viant/toolbox/url"
+	"github.com/viant/afs"
+	"github.com/viant/afs/storage"
+	"github.com/viant/afs/scp"
+	"github.com/viant/afsc/s3"
+	"github.com/viant/afsc/gs"
+)
+
+const sshScheme = "ssh"
+
+var fs = afs.New()
+var fsFaker = afs.NewFaker()
+
+//GetStorageService return toolbox storage service
+func GetStorageService(ctx *endly.Context, resource *url.Resource) (afs.Service, error) {
+	var state = ctx.State()
+	if state.Has(useMemoryService) {
+		return fsFaker, nil
+	}
+	_ = fs.Close(resource.URL)
```

Comment:
what fs.Close retun, why use blank identifier ?

Reply:
you close previously used resources, ignoring potential errors